### DISCUSSION TITLE
Discord Problem

### DISCUSSION THREAD

**Discussion by hubble658** [mylinuxforwork/dotfiles#178] [General]
I completed the installation by following your video but I can't run discord in the app launcher menu or cmd.

I run ' com.discordapp.Discord ' but it didnt work

I used " coredumpctl " to see what fails.
 
the error i get is below.
"
Disabling updates already done
[5:0823/203619.513353:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Discord 0.0.65
[5:0823/203621.400968:ERROR:wayland_connection.cc(200)] Failed to connect to Wayland display: No such file or directory (2)
[5:0823/203621.400993:ERROR:ozone_platform_wayland.cc(265)] Failed to initialize Wayland platform
[5:0823/203621.400998:ERROR:env.cc(258)] The platform failed to initialize.  Exiting.
/app/bin/com.discordapp.Discord: line 17:     5 Segmentation fault      (core dumped) env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/discord/Discord --enable-speech-dispatcher $FLAGS "$@"
2024/08/23 20:36:21 socat[3] W exiting on signal 15
"

I tried to fix it trying this codes.
"
sudo pacman -S dbus
sudo systemctl start dbus.service
sudo systemctl enable dbus.service

flatpak uninstall com.discordapp.Discord
flatpak install flathub com.discordapp.Discord
"
but it didnt solve my issue. How can i fix discord?

-Arch Linux

**Comment by mylinuxforwork** [comment]
@hubble658 Converted your issue to a discussion because it is not Dotfiles related. 
how have you installed discord?

**Reply by hubble658** [reply]
Actually, I didn't install it, it existed. But instead of trying to solve the problem, I tried uninstalling it from flatpakhub and downloading it from yay. The problem is solved now.

- flatpak uninstall com.discordapp.Discord
- yay -S discord

Thx for reply

**Reply by hubble658** [reply]
The other thing I want to know is how can I uninstall this? To remove everything and return to the original arch.

**Comment by mylinuxforwork** [comment]
You can remove the folders from .config and uninstall the packages that you don’t need anymore. A complete rollback to a state before the installation isn’t possible. This is technically impossible.

Only if you made a snapshot with timeshift or snapper.

**Reply by hubble658** [reply]
oh ok. thx.
How do i stop getting updates? Is there some kinda setting about that

**Reply by mylinuxforwork** [reply]
If you don't install the updates then it has no effect on your dotfiles. But check the article in Wiki how to create your own configuration with been safe on updates.
https://github.com/mylinuxforwork/dotfiles/wiki/How-to-customize-the-ML4W-Dotfiles
